### PR TITLE
Team Invitation done

### PR DESCRIPTION
**Pull request by Monti934**
# Pull Requests Checklist
 
Assignee: Monti934
 
- \[x\] code commented?
- \[x\] pre-commit checks passed?
- \[x\] pipelines passed?
 
Reviewer:
 
- \[ \] are there any code smells?
- \[ \] did pipeline pass?
- \[ \] is the code well documented?

### CODE REVIEW

**Review by B1s9l — COMMENTED** [review]
Is good: will need file merging when my branch is ready

File: src/components/popups/TeamSetting.tsx [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+        );
+        setTeamName(foundTeam.name);
+        setTeamDescription(foundTeam.description);
+        setTeamUUID(foundTeam.teamUUID);
+        setInviteURL(`${baseURL}/invitation/${foundTeam.teamUUID}`);
+      } catch (error) {
+        setError("Something went wrong. Please try again");
+        if (error.response.status === 401) {
+          setLeaveError("You are not authorized to do this");
+        }
```
Comment (5v3nn) [inline src/components/popups/TeamSetting.tsx:41]: log unexpected error

File: src/components/popups/TeamSetting.tsx [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+        const foundTeam = response.data.find(
+          (team) => team.teamId === parseInt(teamId)
+        );
+        setTeamName(foundTeam.name);
+        setTeamDescription(foundTeam.description);
+        setTeamUUID(foundTeam.teamUUID);
+        setInviteURL(`${baseURL}/invitation/${foundTeam.teamUUID}`);
+      } catch (error) {
+        setError("Something went wrong. Please try again");
+        if (error.response.status === 401) {
```
Comment (5v3nn) [inline src/components/popups/TeamSetting.tsx:39]: @Monti934 @B1s9l , if 404 and/or 401, is then the user rerouted to their team overview?

File: src/components/popups/TeamSetting.tsx [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+            Authorization: `${token}`,
+          },
+        });
+        setTeamMembers(response.data);
+      } catch (error) {
+        setError("Something went wrong. Please try again");
+        if (error.response.status === 401) {
+          setLeaveError("You are not authorized to do this");
+        }
+      }
```
Comment (5v3nn) [inline src/components/popups/TeamSetting.tsx:60]: log error (otherwise no chance to see full error

File: gitmessage.txt
Comment (5v3nn) [inline gitmessage.txt:1]: please remove

**Review by 5v3nn — CHANGES_REQUESTED** [review]
please add this requested changes

File: src/components/popups/TeamSetting.tsx (hunk 1 above)
Comment (Monti934) [reply src/components/popups/TeamSetting.tsx:41]: done

File: src/components/popups/TeamSetting.tsx (hunk 2 above)
Comment (Monti934) [reply src/components/popups/TeamSetting.tsx:39]: - [ ] will think about this later, at the moment the user would not see the error message anymore
- [ ] maybe add a button with error to go back

File: src/components/popups/TeamSetting.tsx (hunk 3 above)
Comment (Monti934) [reply src/components/popups/TeamSetting.tsx:62]: done

File: gitmessage.txt
Comment (Monti934) [reply gitmessage.txt:1]: deleted

**Review by B1s9l — APPROVED** [review]
Changed or marked all requested problems

**Review by 5v3nn — APPROVED** [review]
perfect, thank you

**Pull request merged**